demisto_sdk/commands/common/logger.py

```diff
@@ -163,24 +297,10 @@ def replace_escapes(self, message):
     file_handler.set_name(FILE_HANDLER)
     file_handler.setLevel(file_log_threshold if file_log_threshold else logging.DEBUG)
 
-    class NoColorFileFormatter(logging.Formatter):
-        def __init__(
-            self,
-        ):
-            super().__init__(
-                fmt="[%(asctime)s] - [%(threadName)s] - [%(levelname)s] - %(filename)s:%(lineno)d - %(message)s",
-                datefmt=DATE_FORMAT,
-            )
-
-        def format(self, record):
-            message = logging.Formatter.format(self, record)
-            message = self.replace_escapes(message)
-            return message
-
-        def replace_escapes(self, message):
-            for key in escapes:
-                message = message.replace(key, "")
-            return message
+    if os.getenv("DEMISTO_SDK_LOG_NO_COLORS", "False") == "True":
```

Comment:
use the `str2bool` function

Reply:
[Done](https://github.com/demisto/demisto-sdk/pull/2994/commits/00c68f386937f56f60f87547d69e4f58ace5ec78).